src/wasm/wasm-emscripten.cpp

```diff
@@ -32,6 +32,7 @@ namespace wasm {
 
 cashew::IString EMSCRIPTEN_ASM_CONST("emscripten_asm_const");
 cashew::IString EM_JS_PREFIX("__em_js__");
+static const char* INVOKE_PREFIX = "invoke_";
```

Comment:
a grep shows there are two more (pre-existing) places in this file that could use this new constant